IDEAS/LIDEAS/Components/BaseClasses/Inputs.mo

```diff
@@ -0,0 +1,9 @@
+within IDEAS.LIDEAS.Components.BaseClasses;
+expandable connector Inputs
```

Comment:
Can we drop the `expandable` here? If so, then I would prefer that to avoid wrong use and to allow the Modelica tools to have easier error checking.

Reply:
Done.